builtin/providers/aws/resource_aws_rds_cluster.go

```diff
@@ -71,6 +71,12 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Computed: true,
 			},
 
+			"storage_encrypted": &schema.Schema{
```

Comment:
The default is `false` if the parameter is omitted from the request, so I see two options. I could either update the schema to include the default value, or I could only set the `StorageEncrypted` value on the `CreateDBClusterInput` object if a value is provided. Which do you think is best?